### PR TITLE
FCT-503: GoodStore presets for `Customers` & `Addresses`

### PR DESCRIPTION
**Pull request by valoriecarli**
This PR contains GoodStore `Customer` & `Address` presets based off of the Customers sheet from [the following workbook](https://docs.google.com/spreadsheets/d/1ehbT8iThVOmS3WdE_3a1eTNlCsSYCVoSOSd1X8hAMDw/edit#gid=463767599).

### CODE REVIEW

**Review by tdeekens — APPROVED** [review]

File: models/commons/src/address/address-draft/presets/sample-data-goodstore/jennifer-jones-address.spec.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
+import { TAddressDraft, TAddressDraftGraphql } from '../../../types';
+// import samSmithAddresses from './sam-smith-addresses';
```
Comment (stephsprinkle) [inline models/commons/src/address/address-draft/presets/sample-data-goodstore/jennifer-jones-address.spec.ts:2]:
```suggestion
```

File: models/commons/src/address/address-draft/presets/sample-data-goodstore/jennifer-jones-address.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { TAddressDraftBuilder } from '../../../types';
+import * as AddressDraft from '../../index';
+
+const jenniferJonesAddress = (): TAddressDraftBuilder =>
```
[the remark this answers is not in the document]
Comment (stephsprinkle) [reply models/commons/src/address/address-draft/presets/sample-data-goodstore/jennifer-jones-address.ts:4]: I genuinely don't have a high care amount on this naming convention. We didn't add `address` in the sample data but I can see it being valuable

[the remark this answers is not in the document]
Comment (jaikamat) [reply models/commons/src/address/address-draft/presets/sample-data-goodstore/jennifer-jones-address.ts:4]: This is helpful naming, especially when we pull these into other models and have to maintain there 👍

File: models/commons/src/address/address-draft/presets/sample-data-goodstore/jennifer-robinson-address.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+const jenniferRobinsonAddress = (): TAddressDraftBuilder =>
+  AddressDraft.presets
+    .empty()
+    .firstName('Jennifer')
+    .lastName('Robinson')
+    .streetName('Second Street')
+    .streetNumber('15')
+    .postalCode('10001')
+    .city('New York City')
```
Comment (stephsprinkle) [inline models/commons/src/address/address-draft/presets/sample-data-goodstore/jennifer-robinson-address.ts:12]:
```suggestion
    .city('New York City')
    .state('NY')
```

File: models/commons/src/address/address-draft/presets/sample-data-goodstore/sebastian-franklin-address.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+const sebastianFranklinAddress = (): TAddressDraftBuilder =>
+  AddressDraft.presets
+    .empty()
+    .firstName('Sebastian')
+    .lastName('Franklin')
+    .streetName('South Road')
+    .streetNumber('9')
+    .postalCode('27517')
+    .city('Durham')
```
Comment (stephsprinkle) [inline models/commons/src/address/address-draft/presets/sample-data-goodstore/sebastian-franklin-address.ts:12]:
```suggestion
    .city('Durham')
    .state('NC')
```

File: models/customer/src/customer-draft/presets/sample-data-goodstore/sebastian-franklin.ts [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import { AddressDraft } from '@commercetools-test-data/commons';
+import { authenticationMode } from '../../../constants';
+import type { TCustomerDraftBuilder } from '../../../types';
+import * as CustomerDraft from '../../index';
+
+const sebastianFranklinAddress = (): TCustomerDraftBuilder =>
```
Comment (jaikamat) [inline models/customer/src/customer-draft/presets/sample-data-goodstore/sebastian-franklin.ts:6]: This one isn't an address, the variable name might need to be adjusted

File: models/commons/src/address/address-draft/presets/sample-data-goodstore/jennifer-jones-address.spec.ts (hunk 1 above)
Comment (tylermorrisford) [inline models/commons/src/address/address-draft/presets/sample-data-goodstore/jennifer-jones-address.spec.ts:2]: ☝️ assuming this is some copy pasta

**Review by stephsprinkle — COMMENTED** [review]
One missing field, the rest looks great!

**Review by jaikamat — APPROVED** [review]
You're an absolute legend 💯

**Pull request merged**